pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -318,6 +318,12 @@ type DashboardSpec struct {
 	// SSL determines whether SSL should be used
 	// +optional
 	SSL bool `json:"ssl,omitempty"`
+	// Prometheus Host end point
+	// +optional
+	PrometheusEndpoint string `json:"prometheusEndpoint,omitempty"`
+	// Prometheus Host end point SSL verify
+	// +optional
+	PrometheusEndpointSSLVerify bool `json:"prometheusEndpointSSLVerify,omitempty"`
```

Comment:
Maybe its better to have a different struct `Settings` that can hold other possible settings of the dashboard in the future as mentioned by @travisn in one of his comments.

Reply:
> Maybe its better to have a different struct `Settings` that can hold other possible settings of the dashboard in the future as mentioned by @travisn in one of his comments.

didn't want to alter the already existing attributes but if it's possible then I think it's a good idea to group the settings.

Reply:
Let's have the setting name that is simplest for the user to discover and understand. In this case, I think a setting directly under the dashboard settings like this is appropriate. If in the future we needed to support more general dashboard settings, then perhaps we could create a "settings" section under dashboard.